.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Docs Deploy'
+
+on:
+  release:
+    types: ['published']
+
+jobs:
+  deploy:
+    name: 'Deploy'
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - run: cd docs
+      - name: Install Packages
+        run: npm install
```

Comment:
I believe that this won't work as expected, because every step starts from the initial directory, so `cd docs` in one step doesn't mean that `npm install` will be run in `docs/` in the following step.

We could use `|` or `working-directory`.

Reply:
Ah interesting. I’ll fix that.